metricbeat/module/aerospike/_meta/config.yml

```diff
@@ -1,2 +1,4 @@
 - module: aerospike
+  metricsets: ["namespace"]
+  period: 10s
```

Comment:
I remember @andrewkroh suggested to remove period. The part I like about it is that it makes the config even shorter.

Reply:
One option would be to only include the period when it's not the default of 10s.

Reply:
In general, I would prefer to keep it as it adds visibility to the setting, any user opening the config file for the module will know what that is and how it works really quick. If we remove it they need to go back to the docs.